### DISCUSSION TITLE
v1.5.1-beta

### DISCUSSION THREAD

**Discussion by albertpb** [albertpb/sd-manager#15] [General]
- Fix memory bug on workers. Force to terminate when finish.
- Fix set checkpoint and lora path not importing models
- Clarify message when found models with same hash

<hr /><em>This discussion was created from the release <a href='https://github.com/albertpb/sd-manager/releases/tag/v1.5.1'>v1.5.1-beta</a>.</em>

**Comment by AsuraAce** [comment]
can´t start 1.5.1

![image](https://github.com/albertpb/sd-manager/assets/13394741/0de5055f-86d6-4fdf-b5d6-2bf9cbfd67dc)

**Reply by albertpb** [reply]
For some reason it works for me. I'll have to bundle that lib in the package maybe.

I'm fixing an issue related to memory on the worker threads now. When I'm done I'll release the next version.
